### ISSUE TITLE
Scrollable min/max date bugs

### ISSUE THREAD

**Issue by majugurci** [BugiDev/react-native-calendar-strip#191]
Hello,
there are a few bugs when scrollable is enabled and both min and max dates are set.
It is happening when minDate is set to the beginning of the current month.
Seems both android and ios are affected.

Bugs: 
- Title is incorrect on initial render
- There is no right arrow (in the provided example left arrow is correctly not shown - you might want to double check for other scenarios).

After you swipe calendar strip is rerendered good.

Image:
![image](https://user-images.githubusercontent.com/481077/83503969-318b9d00-a4c4-11ea-81dc-50144c711351.png)

Reproducible example: [Expo snack](https://snack.expo.io/@mjugurcic/rn-calendar-strip-min_max-date-bug)

**Comment by Tyred** [comment]
I am having the same issue. If you find any workaround or something like that pls let me know :)

**Comment by peacechen** [comment]
Thanks for reporting this.  I'll debug as soon as I can, but if you have time to investigate the CalendarStrip code, please submit a PR or comment here.

**Comment by peacechen** [comment]
The fix has been published in 2.0.2

**Comment by majugurci** [comment]
I have tested these use cases and it is working correctly now. Thank you very much for fast response and fix.

But I have found another bug, I'm guessing it is related to this so I'll post it here instead of opening a new ticket.

Description:
In some cases right arrow is missing and by clicking right arrow you cannot go to the last date.
I hope these images make it clearer:

- Scrolled to the last date:
![image](https://user-images.githubusercontent.com/481077/83732698-b7d2eb00-a64c-11ea-99ad-09ea75390dec.png)

- Scrolled just so last day is not visible = no right arrow
![image](https://user-images.githubusercontent.com/481077/83733310-aa6a3080-a64d-11ea-997b-4f5eeaf3f95c.png)

- And finally scrolled one more day (so 2 days at the end are not visible) = right arrow is visible but pressing it will move just one day, not 2 days as expected
![image](https://user-images.githubusercontent.com/481077/83733540-f3ba8000-a64d-11ea-9b97-6c7da80ea227.png)

Reproducible example: [Expo snack](https://snack.expo.io/@mjugurcic/rn-calendar-strip-scrollable-end-bug)
